### ISSUE TITLE
[Bug] [a11y]PB-Add to homescreen-Touch target needs increase of height for "Cancel" and "Add"

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#9652]
## Prerequisites:
- Have Google Accessibility Scanner installed.

### Steps to reproduce
1. Open app and from the homescreen, tap on "Private Browsing" icon.
2. Tap on the "Add Shortcut" button.
2. Scan the search screen.
### Expected behavior
There are no suggestions to make any changes.
### Actual behavior
Add to homescreen - "Cancel" button
- com.sec.android.app.launcher:id/add_item_cancel_button
This item's height is 40dp. Consider making the height of this touch target 48dp or larger.

Add to homescreen - "Add" button
- com.sec.android.app.launcher:id/add_item_add_button
This item's height is 40dp. Consider making the height of this touch target 48dp or larger.

### Device information

* Android device: Samsung Galaxy S9 (Android 8)
* Fenix version: Firefox Preview Nightly 4/3 (Build #20940609)

**Screenshots:**
**Add to homescreen - "Cancel" button**
![20200403_113815](https://user-images.githubusercontent.com/49859863/78341932-263df300-75a1-11ea-99c9-e4b3cc1dfeac.jpg)

**Add to homescreen - "Add" button**
![Screenshot_20200403-113323_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78341946-2d650100-75a1-11ea-81db-86187e500e35.jpg)

**Comment by mcarare** [comment]
This is a system dialog, so unless we implement one of our own (possibly not worth the work needed), we cannot improve on this. Closing this due to the reason stated.
